deepchem/models/torch_models/scscore.py

```diff
@@ -0,0 +1,53 @@
+import torch
+import torch.nn as nn
+from deepchem.data import NumpyDataset
+from deepchem.feat import CircularFingerprint
+from deepchem.models.torch_models.torch_model import TorchModel
+from deepchem.models.losses import L1Loss
+
+
+class ScScore(nn.Module):
+    """
+    ScScore paper - https://pubs.acs.org/doi/abs/10.1021/acs.jcim.7b00622
```

Comment:
SCScore is more than a feed-forward network. Refer paper for the writing documentation.

Reply:
Sure. I'll recheck.

Reply:
I read the paper. It is a FFNN.

From the paper -
The model, implemented in Python 2.7
using Tensorflow, 30 is a feed forward neural network consisting of five hidden layers with 300
hidden nodes each, ReLU activation with bias, and a single sigmoid output.